<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalQueueInterfaceSettingsWidget</class>
 <widget class="QWidget" name="GlobalQueueInterfaceSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>377</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of seconds to wait before querying the jobs' statuses again with the queue interface.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Queue refresh interval:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Host:</string>
       </property>
       <property name="buddy">
        <cstring>edit_host</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spin_port">
       <property name="prefix">
        <string>SSH Port </string>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>22</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_host">
       <property name="toolTip">
        <string>Address of host. Can use IP or host name.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_description">
       <property name="toolTip">
        <string>Short description of optimization (used as %description% template keyword in input templates).</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_username">
       <property name="toolTip">
        <string>Username on above host.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_20">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Working directory (Server):</string>
       </property>
       <property name="buddy">
        <cstring>edit_rempath</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Working directory (Local):</string>
       </property>
       <property name="buddy">
        <cstring>edit_locpath</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_rempath">
       <property name="toolTip">
        <string>Path on remote host to use during optimizations. Do not use wildcard characters or BASH-specific characters (e.g. '~' in place of /home/user).</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_locpath">
       <property name="toolTip">
        <string>Local path to store files</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QSpinBox" name="spin_interval">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The number of seconds to wait before querying the jobs' statuses again with the queue interface.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="cb_cancelJobAfterTime">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically cancel a job after a specified amount of time. The next pending job will then be submitted.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This option is present primarily because there are sometimes bugs in optimizers that can cause them to run forever. In addition, every once in a while, XtalOpt does not recognize that a job is no longer in the queue; this option prevents that from eventually stopping all jobs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Automatically cancel job after:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>User:</string>
       </property>
       <property name="buddy">
        <cstring>edit_username</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Global Queue Interface Settings&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="buddy">
        <cstring>edit_description</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="spin_hoursForCancelJobAfterTime">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically cancel a job after a specified amount of time. The next pending job will then be submitted.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This option is present primarily because there are sometimes bugs in optimizers that can cause them to run forever. In addition, every once in a while, XtalOpt does not recognize that a job is no longer in the queue; this option prevents that from eventually stopping all jobs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="suffix">
        <string> hours</string>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="3">
      <widget class="QCheckBox" name="cb_logErrorDirs">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Leaves a copy of a directory in which an error occurred in &amp;lt;local_path&amp;gt;/errorDirs.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This can help when figuring out why an xtal fails when submitted.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Only keeps one copy for each xtal directory (so if an error occurred in 00001x00001 twice, the first error directory would have been overwritten by the second)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Log error directories</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="cb_cleanRemoteOnStop">
       <property name="text">
        <string>Clean remote directories when finished</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_host</tabstop>
  <tabstop>spin_port</tabstop>
  <tabstop>edit_username</tabstop>
  <tabstop>edit_rempath</tabstop>
  <tabstop>edit_locpath</tabstop>
  <tabstop>edit_description</tabstop>
  <tabstop>spin_interval</tabstop>
  <tabstop>cb_cleanRemoteOnStop</tabstop>
  <tabstop>cb_logErrorDirs</tabstop>
  <tabstop>cb_cancelJobAfterTime</tabstop>
  <tabstop>spin_hoursForCancelJobAfterTime</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_cancelJobAfterTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_hoursForCancelJobAfterTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
